hs_geographic_feature_resource/receivers.py

```diff
@@ -273,9 +276,10 @@ def geofeature_pre_create_resource(sender, **kwargs):
         else:
             validate_files_dict['are_files_valid'] = False
             validate_files_dict['message'] = "Invalid files uploaded. Please note the three mandatory files (.shp, .shx, .dbf) of ESRI Shapefiles should be uploaded at the same time (or in a zip file)."
-    except:
+    except Exception as ex:
         validate_files_dict['are_files_valid'] = False
         validate_files_dict['message'] = 'Uploaded files are invalid or corrupt.'
+        logger.error("geofeature_pre_create_resource: {0}} ".format(ex.message))
```

Comment:
This error message has unbalanced braces.  Also, consider whether or not `logger.exception(...)` would be appropriate here.